coreos-base/oem-packet/files/reset-interfaces.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
nm, found it: https://github.com/coreos/coreos-overlay/blob/master/coreos-base/oem-rackspace-onmetal/files/rename-interfaces.sh so basically the same